File: Book/book.php
<?php

class Book {
    function __construct($title, $author, $price) {
        $this->title = $title;
        $this->author = $author;
        $this->price = $price;
    }

    function describe() {
        echo "Le livre a pour titre {$this->title}, pour auteur {$this->author} et coûte {$this->price} euros".PHP_EOL;
    }
}


$book1 = new Book("Les chevaliers d'émeraude", "Anne Robillard", 15.30);
$book1->describe();

$book2 = new Book("Le Seigneur des Anneaux", "JRR Tolkien", 17.2);
$book2->describe();

$book3 = new Book("Clean Code", "Robert C. Martin", 24);
$book3->describe();

File: Book/book.py
class Book:
    # title
    # author
    # price

    def __init__(self, title, author, price):
        self.title = title
        self.author = author
        self.price = price

    def describe(self):
        print(f"Le livre a pour titre {self.title}, pour auteur {self.author} et coûte {self.price} euros")


book1 = Book("Les chevaliers d'émeraude", "Anne Robillard", 15.30)
book1.describe()

book2 = Book("Le Seigneur des Anneaux", "JRR Tolkien", 17.2)
book2.describe()

book3 = Book("Clean Code", "Robert C. Martin", 24)
book3.describe()